test/setup-deployment.sh

```diff
@@ -184,7 +184,25 @@ EOF
 EOF
                 ;;
         esac
-        ${KUBECTL} apply --kustomize "$tmpdir/my-deployment"
+        # When quickly taking down one installation of PMEM-CSI and recreating it, we
+        # sometimes we get: nodePort: Invalid value: 32000: provided port is already allocated
```

Comment:
Please remove one 'we' in 'we sometimes we get'.

Reply:
Fixed. Can we merge without waiting for another test round?